docs/en/concepts-and-designs/service-hierarchy.md

```diff
@@ -144,6 +146,22 @@ If you want to customize it according to your own needs, please refer to [Servic
   - VIRTUAL_MQ.service.name: `rocketmq.skywalking-showcase.svc.cluster.local:9876`
   - ROCKETMQ.service.name: `rocketmq::rocketmq.skywalking-showcase`
 
+#### RABBITMQ On K8S_SERVICE
+- Rule name: `short-name`
+- Groovy script: `{ (u, l) -> u.shortName == l.shortName }`
+- Description: RABBITMQ.service.shortName == K8S_SERVICE.service.shortName
+- Matched Example:
+  - RABBITMQ.service.name: `rabbitmq::rabbitmq.skywalking-showcase`
+  - K8S_SERVICE.service.name: `skywalking-showcase::demo-rabbitmq.skywalking-showcase`
+
+#### VIRTUAL_MQ On RABBITMQ
+- Rule name: `lower-short-name-with-fqdn`
+- Groovy script: `{ (u, l) -> u.shortName.substring(0, u.shortName.lastIndexOf(':')) == l.shortName.concat('.svc.cluster.local') }`
+- Description: VIRTUAL_MQ.service.shortName remove port == RABBITMQ.service.shortName with fqdn suffix
+- Matched Example:
+  - VIRTUAL_MQ.service.name: `rabbitmq.skywalking-showcase.svc.cluster.local:5672`
```

Comment:
Is this working for cluster mode of RaabitMQ? Are multiple IPs in this name? Or just DNS because of K8s deployment.

Reply:
k8s dns